src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/__init__.py

```diff
@@ -3,17 +3,24 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+import sys
+import platform
 import argparse
 from collections import OrderedDict
 
 from knack.prompting import prompt_y_n
 from knack.util import CLIError
+from knack.log import get_logger
 
 from azure.cli.core import AzCommandsLoader
 from azure.cli.core.commands import CliCommandType
 
 import azure.cli.command_modules.extension._help  # pylint: disable=unused-import
 
+IS_WINDOWS = sys.platform.lower() in ['windows', 'win32']
+LIST_FILE_PATH = '/etc/apt/sources.list.d/azure-cli.list'
```

Comment:
Use `os.path.join('etc', 'apt', etc.)` instead.
Don't want to hardcode `/`

Reply:
done